02-RTB/verbali-interni/2024-11-04.typ

```diff
@@ -0,0 +1,249 @@
+
+Il gruppo prevede che nel prossimo futuro il cambiamento dei ruoli *non potrà sicuramente avvenire prima di 14 giorni*, minimo periodo per produrre avanzamenti concreti, impossibili da realizzare con un periodo di tempo più corto.
+
+#pagebreak()
+
+== Migliorie alla parte pubblica del repository
+
+Il gruppo accoglie il suggerimento riguardante l'apertura delle risorse in una scheda dedicata: decide perciò di modificare la parte pubblica del repository (sito web) quanto prima per ottenere questo effetto.
+
+Per migliorare ulteriormente l'organizzazione del repository, stabilisce altresì di inserire tutti i nuovi documenti prodotti dal giorno 04-11-2024 sino al superamento della Requirements and Tecnology Baseline (RTB) in una sezione appositamente realizzata: la parte pubblica del repository (sito web) dovrà riflettere questo cambiamento.
+
+#pagebreak()
+
+== Importanti cambiamenti nella redazione dei verbali
+
+_ALimitedGroup_ comprende l'importanza che non è possibile modificare nuovamente un verbale o un documento senza prima che questo venga verificato e comprende anche la necessità di tracciare le decisioni e le attività che seguono le riunioni.\
+Per questo ed altri motivi a carattere pricipalmente migliorativo, il gruppo ha deciso che effetterà degli importanti cambiamenti alla struttura e al metodo di redazione dei verbali, qui di seguito indicata.
+
+=== Modifiche alla struttura dei verbali
+
+Nel corso della redazione dei verbali, il gruppo ha determinato che il commento iniziale stabilito dal *verbale interno* del #link("https://alimitedgroup.github.io/VI%2023-10-2024%201.0.0.pdf")[*23-10-2024*] alla *sezione 2.4* non risulta essere di alcuna utilità: lo sviluppo del repository ha rimosso l'esigenza di inserirlo nel documento grazie a meccanismi di querying automatici forniti da typst.
+Ha tuttavia trovato molto positiva la nomenclatura adottata: per questo motivo il gruppo ha deciso sarà *mantenuta* con lo scopo di nominare i file prodotti. \ \
+
+Per soddisfare l'esigenza di verificare un verbale subito a seguito di una modifica, il registro delle modifiche conterrà ora le seguenti colonne:
+
+#table(
+  columns: (auto, auto, 0.5fr, 0.5fr, 1fr),
+  inset: 5pt,
+  align: center,
+  table.header(
+    text(12pt, fill: black)[*Vers.*],
+    text(12pt, fill: black)[*Data*],
+    text(12pt, fill: black)[*Autore*],
+    text(12pt, fill: black)[*Verificatore*],
+    text(12pt, fill: black)[*Descrizione*],
+  ),
+)
+
+Con la data che indica il momento in cui la modifica effetuata dall'*autore* è stata verificata dal *verificatore*
+
+Il gruppo ha anche stabilito delle modifiche relative al metodo di stesura dei verbali: si veda la @metodo_stesura più avanti in questo documento. \
```

Comment:
Correzione: riga 81 "metodo di stesura dei verbali ~~:si veda la @metodo_stesura più avanti in questo documento~~ , contenute nella @metodo_stesura di questo documento."